www/routes/docs/[...slug].tsx

```diff
@@ -161,17 +161,20 @@ export default function DocsPage(props: PageProps<Data>) {
       <div class="flex flex-col min-h-screen mx-auto max-w-screen-2xl">
         <div class="flex-1 " f-client-nav>
           <div class=" md:flex">
-            <nav class="w-[18rem] flex-shrink-0 hidden md:block px-4">
-              <DocsSidebar
-                versionLinks={page.versionLinks}
-                selectedVersion={page.version}
-              />
-            </nav>
-            <Partial name="docs-main">
-              <div class="w-full min-w-0">
-                <Header title="docs" active="/docs" />
-                <main class="mt-4 min-w-0 mx-auto">
-                  <MobileSidebar page={page} />
+            {!props.isPartial && (
+              <nav class="w-[18rem] flex-shrink-0 hidden md:block px-4">
+                <DocsSidebar
+                  versionLinks={page.versionLinks}
+                  selectedVersion={page.version}
+                />
+              </nav>
```

Comment:
What happens to the rendering of this element if `props.partial` is true? Ditto elsewhere. Excuse me if I'm missing context.

Reply:
It's not changing if we're staying on the docs page, so there's no point to recompute it. Without this change it's recomputed, but then ignored by the partials code in the client. The whole point of this change is to stop computing stuff the client will ignore.

Reply:
Oh! Yep. Makes total sense.